build/teamcity/cockroach/ci/builds/build_docker_image.sh

```diff
@@ -44,4 +46,14 @@ docker save "$docker_tag" | gzip > "${artifacts}/${docker_image_tar_name}".gz
 
 cp upstream_artifacts/cockroach "${artifacts}"/cockroach
 
+sed "s,@FROM@,${docker_tag}," build/deploy/Dockerfile.nonroot.in > build/deploy/Dockerfile.nonroot
+cat build/deploy/Dockerfile.nonroot
```

Comment:
I'm not sure if I understand here correctly and just ask out of curiosity -- does it mean that `docker build` will look at both `Dockerfile` and `Dockerfile.nonroot`? And the `cat build/deploy/Dockerfile.nonroot`, is it just for logging/debugging purpose, or it's necessary for `docker build` to know this file?

Reply:
`cat` is for logging purposes only.
`docker build` will build 2 images using `Dockerfile` and `Dockerfile.nonroot`.
Does it make sense?

Reply:
Ah I got it, so `cockroachdb/cockroach-ci-nonroot` is built on top of `cockroachdb/cockroach-ci`, with additional specification from `Dockerfile.nonroot`. Thanks for explaining!!